modules/Surveys/Lines/Lines.php

```diff
@@ -27,7 +27,7 @@ function ($a, $b) {
     );
     $smarty->assign('questions', $questions);
     $smarty->assign('message', '');
-    if ($view == 'EditView') {
+    if ($view == 'EditView' && $_REQUEST['isDuplicate'] === false) {
```

Comment:
Hi @serhiisamko091184, we need  to check if `$_REQUEST['isDuplicate']` is set, otherwise we can get warnings/notices. Something like:

```
$isDuplicate = $_REQUEST['isDuplicate'] ?? false
if ($view == 'EditView' && isDuplicate === false) {
```

Reply:
Thanks for feedback @clemente-raposo ,
I did requested changes. 
Thanks a lot!

Regards,
Serhii